src/owl.Router.js

```diff
@@ -34,40 +34,40 @@
                 return owl.Promise.resolve(null);
             }
 
-            return this.resolve(route).then(function (ret) {
-              return ret ? that.run(path, route) : null;
-            })
+            return this.resolve(route).then(function (resolveResult) {
+                const calledAllResolves = resolveResult.every(function (a) {return a;});
+                return calledAllResolves ? that.run(path, route, resolveResult) : null;
+            }).catch(function (e) {
+                console.error('Error in Router.open', e.message, e.stack);
```

Comment:
Let's call defaultRoute here and pass error as second parameter to make possible to process errors.